core/events/events_block_base.ts

```diff
@@ -32,7 +34,7 @@ export class BlockBase extends AbstractEvent {
    */
   constructor(opt_block?: Block) {
     super();
-    this.isBlank = !!opt_block;
+    this.isBlank = !opt_block;
```

Comment:
Can we have a comment about this property?

Reply:
It has a docs comment [here](https://github.com/google/blockly/blob/6122ab22c4207d30681dfb6a3a96588419a9b980/core/events/events_abstract.ts#L28), do you think that's sufficient? I don't necessarily want to duplicate docs that might go out-of-date.

Reply:
Oh, cool.  Agreed, don't want to duplicate comments.

That said, that comment looks like Copilot wrote it.  `X.isY` -> "Whether or not X is Y" 🙄  I still have no idea what a 'blank' event means.

Reply:
Updated. How's that feel?

Reply:
Awesome!